src/services/anchor-service.ts

```diff
@@ -468,6 +468,7 @@ export class AnchorService {
       await queryRunner.release()
     }
 
+    Metrics.count(METRIC_NAMES.ACCEPTED_REQUESTS, acceptedRequests.length)
```

Comment:
just a suggestion: put this under https://github.com/ceramicnetwork/ceramic-anchor-service/blob/5d19f0ea60a5af5e0be3ca756322e0a6d97bb157/src/services/anchor-service.ts#L225

Reply:
I don't think those are necessarily the same thing?  I wanted counts at each stage in the process just in case?